src/android/src/main/java/org/pgsqlite/SQLitePlugin.java

```diff
@@ -357,6 +357,7 @@ private void startDatabase(String dbname, JSONObject options, CallbackContext cb
     private SQLiteDatabase openDatabase(String dbname, String assetFilePath, int openFlags, CallbackContext cbc) throws Exception {
```

Comment:
There is also a parallel native implementation for Android SQLitePlugin but seems not to be included in your PR at all?